### PR TITLE
Fix Spotify OS requirement and stanza order

### PR DESCRIPTION
**Pull request by iniquities**

### CODE REVIEW

**Comment by adityadalal924** [comment]
Thank you for the contribution. It was merged directly as 4713c5b866e870910ed502a3da86e87f382537db to keep commit history cleaner. Your contribution is still credited to you.

File: Casks/spotify.rb [hunk 1]
```diff
@@ -6,7 +6,8 @@
   name 'Spotify'
   homepage 'https://www.spotify.com'
   license :gratis
-  depends_on :macos => '>= :snow_leopard'
+
```
Comment (vitorgalvao) [inline Casks/spotify.rb]: There’s trailing whitespace here, which will make the build fail.

**Comment by vitorgalvao** [comment]
Thank you for this. The stanza order rule is pretty new, so many casks will have errors like this. Any fix is appreciated.

File: Casks/spotify.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/spotify.rb]: No seat, fixed in another commit.

**Pull request closed without merging**